Source/UsercentricsAdapter.swift

```diff
@@ -136,7 +136,7 @@ public final class UsercentricsAdapter: NSObject, ConsentAdapter {
     /// Sets up the module to make it ready to be used.
     /// - completion: A completion handler to be executed when the module is done initializing.
     /// An error should be passed if the initialization failed, whereas `nil` should be passed if it succeeded.
-    public func initialize(completion: @escaping (Error?) -> Void) {
+    public func initialize(configuration: ModuleInitializationConfiguration, completion: @escaping (Error?) -> Void) {
```

Comment:
Can you also update the comments to match the ones in the original method definition on the SDK?
Also if you want you can update the changelog and min Core SDK required version in the podspec now so we don't forget.